src/scripts/scenes/xr-scene.js

```diff
@@ -152,24 +151,24 @@ export default class XrScene {
       this.scene.add(controls.getObject());
     }
   }
-
-  _translateViewMatrix(viewMatrix, position) {
-    // Save initial position for later
-    const tempPosition = new Vector3(position.x, position.y, position.z);
-    const tempViewMatrix = new Matrix4().copy(viewMatrix);
-
-    tempViewMatrix.setPosition(new Vector3());
-    tempPosition.applyMatrix4(tempViewMatrix);
-
-    const translationInView = new Matrix4();
-    translationInView.makeTranslation(
-      tempPosition.x,
-      tempPosition.y,
-      tempPosition.z
-    );
-
-    viewMatrix.premultiply(translationInView);
-  }
+  //
+  // _translateViewMatrix(viewMatrix, position) {
+  //   // Save initial position for later
+  //   const tempPosition = new Vector3(position.x, position.y, position.z);
+  //   const tempViewMatrix = new Matrix4().copy(viewMatrix);
+  //
+  //   tempViewMatrix.setPosition(new Vector3());
+  //   tempPosition.applyMatrix4(tempViewMatrix);
+  //
+  //   const translationInView = new Matrix4();
+  //   translationInView.makeTranslation(
+  //     tempPosition.x,
+  //     tempPosition.y,
+  //     tempPosition.z
+  //   );
+  //
+  //   viewMatrix.premultiply(translationInView);
+  // }
```

Comment:
Let's just get rid of that code rather than letting it commented. One can git log to find it back

Reply:
Removed in updated history.